### ISSUE TITLE
Typo results in an error: p { width: 3: }

### ISSUE THREAD

**Issue by tanertopal** [postcss/postcss-safe-parser#6]
I have an issue where a typo results in an error. CSS example:

```
p {
  width: 3:
}
```

In the CSS instead of a semicolon a colon was used to end the line. Is this even supposed to be covered by the safe parser? Not sure if this falls under the bug category :|

**Comment by ai** [comment]
Yeap, Safe parser should not throw a error.

Could you write full stacktrace.

But I will look deeply into it only in next month (need to prepare for talk).

**Comment by tanertopal** [comment]
Ok – maybe I'll have a look till than and make a PR but if not happy to hear is the error output:

```
LazyResult {
          stringified: false,
          processed: false,
          error:
           TypeError: Cannot read property '0' of undefined
               at SafeParser.decl (/node_modules/postcss/lib/parser.es6:182:17)
               at SafeParser.precheckMissedSemicolon (/source/safe-parser.es6:63:14)
               at SafeParser.decl (/node_modules/postcss/lib/parser.es6:215:14)
               at SafeParser.other (/node_modules/postcss/lib/parser.es6:149:18)
               at SafeParser.loop (/node_modules/postcss/lib/parser.es6:55:22)
               at safeParse (/source/safe-parse.es6:10:12)
               at new LazyResult (/node_modules/postcss/lib/lazy-result.es6:42:24)
               at Processor.process (/node_modules/postcss/lib/processor.es6:95:16)
               at /src/index.js:306:75,
          result:
           Result {
             processor: Processor { version: '5.2.5', plugins: [Object] },
             messages: [],
             root: undefined,
             opts: { parser: [Function: safeParse] },
             css: undefined,
             map: undefined } }
```

**Comment by ai** [comment]
Fixed b254293

I will release, when Travis CI will fix their current `nvm` issue.

**Comment by ai** [comment]
Released in 2.0.1

**Comment by tanertopal** [comment]
Thanks a lot! Highly appreciated!
